content/ja/docs/reference/glossary/api-eviction.md

```diff
@@ -0,0 +1,19 @@
+---
+title: APIを起点とした退避
+id: api-eviction
+date: 2021-04-27
+full_link: /ja/docs/concepts/scheduling-eviction/api-eviction/
+short_description: >
+  APIを起点とした退避は、退避APIを使用してEvictionオブジェクトを作成し、Podの正常終了を起動させるプロセスです。
+aka:
+tags:
+- operation
+---
+APIを起点とした退避は、[退避API](/docs/reference/generated/kubernetes-api/{{<param "version">}}/#create-eviction-pod-v1-core)を使用して退避オブジェクトを作成し、Podの正常終了を起動させるプロセスです。
+
+
+<!--more-->
+
+`kubectl drain`コマンドのようなkube-apiserverのクライアントを使用し、退避APIを直接呼び出すことで、退避を要求することができます。`退避`オブジェクトが生成された時、APIサーバーは対象のPodを終了させます。
```

Comment:
> APIサーバーは対象のPodを終了させます

Perhaps a suggestion to upstream document.
I found it difficult to understand the difference between this and the DELETE API.
The Eviction API complies with PodDisruptionBudget.

Since, I think [`PodDisruptionBudgets`](/docs/tasks/run-application/configure-pdb/) is so important.